wandb/proto/wandb_telemetry.proto

```diff
@@ -21,6 +21,7 @@ message TelemetryRecord {
   Issues issues = 11;
   string core_version = 12;
   string platform = 13;
+  string launch_run_queue_item_id = 14;
```

Comment:
why do you put it in telemetry?

Reply:
I agree with Katia. Telemetry is used for analytics, it tracks feature uses, imports etc. We can inject this directly into the _wandb key in the config.